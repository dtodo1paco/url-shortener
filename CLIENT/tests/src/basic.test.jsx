import React from 'react';
import { shallow, mount } from 'enzyme';
import RouterApp from 'views/MainApp.jsx';

test('it should render', () => {
    shallow(<RouterApp />);
});

// snapshot test - test the final HTML
test('it should render the expected HTML', () => {
    expect(
    mount(<RouterApp />).html()
).toMatchSnapshot();
});
