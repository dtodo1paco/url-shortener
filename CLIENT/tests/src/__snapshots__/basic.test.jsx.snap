exports[`test it should render the expected HTML 1`] = `"<div><div class=\"md-paper md-paper--1 md-card md-background--card main-form md-block-centered\"><section class=\"md-media md-media--16-9\"><img src=\"http://inspirationseek.com/wp-content/uploads/2014/06/Abstract-Painting-Ideas.jpg\" role=\"presentation\"><div class=\"md-media-overlay\"><div class=\"md-card-title md-card-title--primary\"><div class=\"md-card-title--title-block\"><h2 class=\"md-card-title--title md-card-title--large md-text\">URL Shortener</h2><h3 class=\"md-card-title--title md-text--secondary\">Type a URL and click \'Shorten\'</h3></div></div></div></section><form class=\"md-grid\"><div class=\"url-form\"><div class=\"md-text-field-container md-full-width md-text-field-container--input md-cell md-cell--9 md-cell--3-phone md-cell--6-tablet\"><label for=\"floating-center-title\" class=\"md-floating-label md-floating-label--inactive md-floating-label--inactive-sized md-text--secondary\">URL *</label><input type=\"text\" value=\"\" required=\"\" id=\"floating-center-title\" class=\"md-text-field md-text-field--floating-margin md-full-width md-text\"><hr class=\"md-divider md-divider--text-field md-divider--expand-from-center\"><div class=\"md-text-field-message-container md-text-field-message-container--count-only md-full-width md-text--disabled\"><div aria-hidden=\"true\" class=\"md-text-field-message md-text-field-message--inactive\">Please, type a URL before click button</div><!-- react-empty: 18 --></div></div><button type=\"button\" class=\"md-btn md-btn--flat md-btn--text md-pointer--hover md-text md-inline-block md-cell md-cell--2 md-cell--1-phone md-cell--2-tablet\"><div class=\"md-ink-container\"></div><!-- react-text: 21 -->Shorten<!-- /react-text --></button></div></form></div><footer><span><a href=\"https://dtodo1paco.github.io/cv/fjap.html\"><!-- react-text: 25 -->@<!-- /react-text --><!-- react-text: 26 -->dtodo1paco<!-- /react-text --></a></span></footer></div>"`;
